cypress/e2e/po/pages/extensions.po.ts

```diff
@@ -228,6 +229,14 @@ export default class ExtensionsPagePo extends PagePo {
     return this.extensionTabs.clickNthTab(4);
   }
 
+  extensionTabBuiltinClick(): Cypress.Chainable {
```

Comment:
if doesn't actually give us much more, but for consistency in case of future changes this should use `TabbedPo` and `clickTabWithSelector('[data-testid="btn-builtin"]')` 

the original PO should have implemented a clickTabWithName(name) --> `this.self().find(btn-${ name }).click())` kinda method

Reply:
Addressed